src/Database/Dialect/MysqlDialectTrait.php

```diff
@@ -81,4 +81,12 @@ public function enableForeignKeySQL()
     {
         return 'SET foreign_key_checks = 1';
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function explainSQL($query)
```

Comment:
Should the method be `explainSql`?
We never uppercase abbr, it is also JsonView, Mysql class etc.